src/pages/TransactionDuplicate/ReviewTaxCode.tsx

```diff
@@ -25,30 +26,41 @@ function ReviewTaxRate() {
     const compareResult = TransactionUtils.compareDuplicateTransactionFields(transactionID);
     const stepNames = Object.keys(compareResult.change ?? {}).map((key, index) => (index + 1).toString());
     const {currentScreenIndex, navigateToNextScreen} = useReviewDuplicatesNavigation(Object.keys(compareResult.change ?? {}), 'taxCode', route.params.threadReportID ?? '');
+    const transaction = TransactionUtils.getTransaction(transactionID);
     const options = useMemo(
         () =>
             compareResult.change.taxCode?.map((taxID) =>
                 !taxID
-                    ? {text: translate('violations.none'), value: undefined}
+                    ? {text: translate('violations.none'), value: TransactionUtils.getDefaultTaxCode(policy, transaction) ?? ''}
                     : {
                           text: PolicyUtils.getTaxByID(policy, taxID)?.name ?? '',
                           value: taxID,
                       },
             ),
-        [compareResult.change.taxCode, policy, translate],
+        [compareResult.change.taxCode, policy, transaction, translate],
+    );
+    const getTaxAmount = useCallback(
+        (taxID: string) => {
+            const taxPercentage = TransactionUtils.getTaxValue(policy, transaction, taxID);
+            return CurrencyUtils.convertToBackendAmount(TransactionUtils.calculateTaxAmount(taxPercentage ?? '', TransactionUtils.getAmount(transaction), transaction?.currency ?? ''));
+        },
+        [policy, transaction],
     );
 
-    const onSelectRow = (data: FieldItemType) => {
-        if (data.value !== undefined) {
-            setReviewDuplicatesKey({taxCode: data.value as string});
-        }
-        navigateToNextScreen();
-    };
+    const onSelectRow = useCallback(
```

Comment:
This should say what it does not what it handles, but just ignore it for now since it's too many changes 😄